src/components/Events/SingleEvent.vue

```diff
@@ -0,0 +1,97 @@
+<template>
```

Comment:
Can you add screen shot of what the page looks like?

Reply:
<img width="1029" alt="Screen Shot 2020-02-05 at 11 40 12 PM" src="https://user-images.githubusercontent.com/55663537/73906645-a3830b00-4871-11ea-8c52-fb0c81028f4e.png">